src/lib/components/forms/label-input.svelte

```diff
@@ -1,31 +1,54 @@
 <script>
+  export let value = '';
   export let label = '';
   export let id = '';
   export let type = 'text';
   export let placeholder = '';
   export let isTextArea = false;
+  export let required = false;
   export let horizontal = false;
+  export let textGap;
+  export let name = '';
 </script>
 
-<div class="flex flex-{horizontal ? 'row' : 'col'}">
+<div
+  class="flex flex-{horizontal ? 'row justify-between' : 'col'} {$$props.className}"
+  class:align-middle={horizontal && !isTextArea}
```

Comment:
```suggestion
  class="flex flex-{horizontal ? 'row' : 'col'} {$$props.className}"
```

From my tests this works the same just simpler, but lmk if you need them there for any reason!